$NetBSD: patch-ad,v 1.2 2015/12/29 23:34:55 dholland Exp $

Use sdbm instead of ndbm on BSD. XXX: does not match the pkgsrc makefile.

--- ndbm.c	2002/08/12 16:08:46	1.1
+++ ndbm.c	2002/08/12 16:10:13
@@ -1,7 +1,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/param.h>
+#if (defined(BSD) && BSD >= 199306)
+#include <sdbm.h>
+#else
 #include <ndbm.h>
+#endif
 #include "siod.h"
 #if defined(sun) || defined(hpux) || defined(linux) || defined(sgi)
 #include <sys/types.h>
